packages/core/http/core-http-server/src/versioning/types.ts

```diff
@@ -55,17 +55,23 @@ export type VersionedRouteConfig<Method extends RouteMethod> = Omit<
   enableQueryVersion?: boolean;
 
   /**
-   * Human-friendly description of this route, should be usable for documentation
+   * Human-friendly summary of this route, used for (OAS) documentation
```

Comment:
```suggestion
   * Human-friendly summary of this route, used for openAPI documentation
```
With a link to the docs

Reply:
Good catch! I'll make these the same as we have for the router interface!